e2e/api/steps.js

```diff
@@ -117,8 +117,7 @@ module.exports = {
 
     let createClaimData = data.CREATE_CLAIM(mpScenario);
     // Remove after court location toggle is removed
-    createClaimData = await replaceWithCourtNumberIfCourtLocationDynamicListIsNotEnabled(createClaimData);
-
+    //createClaimData = await replaceWithCourtNumberIfCourtLocationDynamicListIsNotEnabled(createClaimData);
```

Comment:
if court location toggle is removed, then this line should be removed instead of commented out; otherwise, it should stay

Reply:
Will remove it

Reply:
Now removed